Cargo.toml

```
@@ -18,14 +18,14 @@ include = [
     "/LICENSE-MIT",
     "/LICENSE-APACHE",
 ]
-edition = "2018"
+edition = "2021"
 
 [dependencies]
-embedded-hal = "0.2"
+embedded-hal = "1.0"
 
 [dev-dependencies]
-linux-embedded-hal = "0.3"
-embedded-hal-mock = "0.8"
+linux-embedded-hal = "0.4"
+embedded-hal-mock = "0.10"
```

```suggestion
embedded-hal-mock = { version="0.10", default-features=false, features=["eh1"] }
```
This way we avoid the dependency on `embedded-hal` 0.2